eleventy.config.cjs

```diff
@@ -57,6 +58,7 @@ module.exports = (eleventyConfig) => {
   );
 
   eleventyConfig.addPlugin(EleventyRenderPlugin);
+  eleventyConfig.addPlugin(syntaxHighlight);
```

Comment:
Under the hood this is using [Prism](https://prismjs.com/) for syntax highlighting.